scripts/example_relationships_establishments.py

```diff
@@ -1,8 +1,79 @@
 import json
+import re
+import pdb
 
+def extract_function_name(line):
+    functions = []
+    # Search for patterns which indicate function calls
+    results = re.findall("(.\w+\s|.\w+)[(]", line)
```

Comment:
what are results make it the variable name.. also do this for the other python function in categorizer.